doc-content/enterprise-only/employee-rostering/er-deploy-script-proc.adoc

```diff
@@ -25,14 +25,16 @@ You can deploy the {PLANNER} Employee Rostering starter application to {OPENSHIF
 If the current OpenShift user name contains characters that are not letters or numbers, the deployment might fail. You can resolve this issue by providing an additional parameter: `./provision.sh setup employee-rostering --binary --project-suffix optaweb`.  This example uses "optaweb", but you can use any value for the suffix.
```

Comment:
Just noting that I have never used the provision.sh script, so I cannot confirm the correctness of the docs regarding it.